package/fetch/src/fetch.ts

```diff
@@ -0,0 +1,139 @@
+import {createLogger, vatrRegisteredList} from '@vatr/logger';
+
+const log = createLogger('vatr/fetch');
+// const error = createLogger('vatr/fetch', 'error', true);
+
+vatrRegisteredList.push({
+  name: '@vatr/fetch',
+  version: '{{VATR_VERSION}}',
+});
+
+declare global {
+  // Patch typescript's global types
+  interface AbortController {
+    abort(reason?: string): void;
+  }
+  interface AbortSignal {
+    reason?: string;
+  }
+}
+
+// @TODO: docs for all options
+export interface FetchOptions extends RequestInit
+{
+  /**
+   * @default 10_000 ms
+   */
+  timeout?: number;
+  bodyObject?: Record<string | number, unknown>;
```

Comment:
I suggest you make a correct type
And export it

Reply:
?!

Reply:
make a type as variable